pkg/controller/traceflow/controller.go

```diff
@@ -265,6 +265,11 @@ func (c *Controller) syncTraceflow(traceflowName string) error {
 }
 
 func (c *Controller) startTraceflow(tf *crdv1alpha1.Traceflow) error {
+	if err := c.validateTraceflow(tf); err != nil {
+		klog.Errorf("Invalid traceflow request %+v, err: %+v", tf, err)
```

Comment:
```suggestion
		klog.ErrorS(err, "Invalid Traceflow request", "request", tf)
```
please use structured logging for all new log messages